validator/validator-main.protoascii

```diff
@@ -4422,7 +4422,10 @@ attr_lists: {
   attrs: { name: "accept" }
   attrs: { name: "accesskey" }
   attrs: { name: "autocomplete" }
-  attrs: { name: "autofocus" }
+  attrs: {
+    disabled_by: "amp4email"
+    name: "autofocus"
```

Comment:
this is a regression, have we verified this does not make amp4email docs invalid?

Reply:
I think this change is ok. `autofocus` is already banned for other elements in AMP4EMAIL e.g. `<select>`.

/cc @ampproject/wg-amp4email